backuper/modules/cloud/amazon/rds.py

```diff
@@ -12,251 +12,176 @@ class ValidateRDS(ValidateBase):
     def params_validate(self, **kwargs):
 
         if kwargs['action'] == 'create':
-
             parameters_schema = self.tr.Dict({
-                self.tr.Key('snapshot_identifier'): self.tr.String,
-                self.tr.Key('region'): self.tr.Enum(*amazon_regions),
-                self.tr.Key('db_identifier'): self.tr.String,
-                self.tr.Key('copy_to_region', optional=True): self.tr.List(
-                    self.tr.String, min_length=1),
-                self.tr.Key('wait_timeout', optional=True): self.tr.Int
+                self.tr.Key('region'): self.tr.Enum(*amazonRegions),
+                self.tr.Key('engine'): self.tr.String,
```

Comment:
engine should not be just a string
it is a list with allowed values (like postgresql, mysql and etc...)

Reply:
done. created
engine: {rds:[], ...}